src/main/java/LineTokenizer.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+class LineTokenizer {
+
+    private final Map<String, List<Integer>> wordsChart;
+
+    public LineTokenizer() {
+        this.wordsChart = new HashMap<>();
```

Comment:
We can move this directly to field declaration

Reply:
moved